tensorboard/data/provider.py

```diff
@@ -307,6 +323,35 @@ def read_blob(self, blob_key):
         pass
 
 
+class ExperimentMetadata(object):
```

Comment:
Just curious, is the thought that we'll have some custom code to serialize this into the JSON XHR response?  Or should this class itself define basically a way to convert it into a dict that can then be converted to JSON?

Reply:
Good question. The thinking is that

1. Defining a data class like this spells out what the experiment metadata is expected to look like.
2. It can potentially have methods like `to_json()` to support convenient conversion to a serializable format.
3. `Run` is also a class in the same file. So this seems nice and consistent.